examples/ingest/elasticsearch/wiki_movie_plots_small.csv

```diff
@@ -0,0 +1,100 @@
+Release Year,Title,Origin/Ethnicity,Director,Cast,Genre,Wiki Page,Plot
+1901,Kansas Saloon Smashers,American,Unknown,,unknown,https://en.wikipedia.org/wiki/Kansas_Saloon_Smashers,"A bartender is working at a saloon, serving drinks to customers. After he fills a stereotypically Irish man's bucket with beer, Carrie Nation and her followers burst inside. They assault the Irish man, pulling his hat over his eyes and then dumping the beer over his head. The group then begin wrecking the bar, smashing the fixtures, mirrors, and breaking the cash register. The bartender then sprays seltzer water in Nation's face before a group of policemen appear and order everybody to leave.[1]"

```

Comment:
I think we could chop this down to say the top 10 or so?

Reply:
Sounds good 👍, I was also having doubts on the exact number of these

Reply:
Changes applied! (chopped down to 31 examples, which leaves 10 docs after processing. [that reduction in numbers should be due to dropna() in pandas])

Reply:
perfect, thanks!